src/xdocs/google_style.xml

```diff
@@ -399,7 +399,7 @@
                             <td><a name="3.4.2.1"/><a href="#3.4.2.1"><img src="images/anchor.png" alt=""/></a></td>
                             <td>
                                 <a
-                                    href="http://checkstyle.sourceforge.net/reports/google-java-style.html#s3.4.2.1-overloads-never-split">3.4.2.1 Overloads: never split</a>
+                                    href="http://checkstyle.sourceforge.net/reports/google-java-style-20160712.html#s3.4.2.1-overloads-never-split">3.4.2.1 Overloads: never split</a>
                             </td>
```

Comment:
Clarified that not even private members can appear between two overloads. OverloadMethodsDeclarationOrder behaves correctly